src/main/java/dev/jbang/catalog/Alias.java

```diff
@@ -21,19 +21,27 @@
 	@SerializedName(value = "java-options")
 	public final List<String> javaOptions;
 	public final Map<String, String> properties;
+	@SerializedName(value = "java", alternate = { "java-version" })
```

Comment:
why the alternate names? shouldn't we not just use the same as long form command line flag ?

Reply:
no special reason, if you don't like them I'll remove them :-)

Reply:
Less ways to configure. Less bugs :) so yes please. Let's keep it clean.

Reply:
Done